inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/SinkFieldBase.java

```diff
@@ -56,7 +56,7 @@ public class SinkFieldBase {
     private String partitionStrategy;
 
     @ApiModelProperty("Extra Param in JSON style")
-    private String extrParam;
+    private String extParam;
```

Comment:
It is recommended to change to `extParams`, which is consistent with the naming in other entities.